src/utils/constants.ts

```diff
@@ -109,12 +110,18 @@ export const ROOT_TYPES = rootTypes;
 /**
  * The Polymesh RPC node version range that is compatible with this version of the SDK
  */
-export const SUPPORTED_NODE_VERSION_RANGE = '5.2.0';
+export const SUPPORTED_NODE_VERSION_RANGE = '5.3.0';
```

Comment:
I think we should have two variables, `MAX_SUPPORTED_NODE_VERSION` and `MIN_SUPPORTED_NODE_VERSION`.

For the first 5.3 release we leave MIN at 5.2 and set MAX to 5.3. Then when we add NFT features we bump the MIN to 5.3.